python/run-tests

```diff
@@ -27,6 +27,8 @@ cd "$FWDIR/python"
 FAILED=0
 LOG_FILE=unit-tests.log
 
+. "$FWDIR"/bin/load-spark-env.sh
```

Comment:
I think that it's being used to calculate `SPARK_SCALA_VERSION` in order to know which JAR to search for.